palette/moreland/convert_test.go

```diff
@@ -24,12 +24,26 @@ func TestRgb_sRGBA(t *testing.T) {
 	}
 	for i, tc := range testCases {
 		result := tc.l.sRGBA(0)
-		if result != tc.s {
+		if sRGBADifferent(result, tc.s, 1.0e-14) {
 			t.Errorf("case %d: have %+v, want %+v", i, result, tc.s)
 		}
 	}
 }
 
+func sRGBADifferent(a, b sRGBA, tolerance float64) bool {
+	return different(a.R, b.R, tolerance) ||
+		different(a.G, b.G, tolerance) ||
+		different(a.B, b.B, tolerance) ||
+		different(a.A, b.A, tolerance)
+}
+
+func different(a, b, tolerance float64) bool {
```

Comment:
Please use "equal" rather than "different"; embedded negation is difficult to read in complex cases (not here, but it makes a general principle).

The function would be the following with the test condition inverted.

```
func sRGBAequalWithin(a, b sRGBA, tol float64) bool {
    return floats.EqualWithinAbsOrRel(a.R, b.R, tol, tol) &&
        floats.EqualWithinAbsOrRel(a.G, b.G, tol, tol) &&
        floats.EqualWithinAbsOrRel(a.B, b.B, tol, tol) &&
        floats.EqualWithinAbsOrRel(a.A, b.A, tol, tol)
}
```